src/styles/blocks/container.scss

```
@@ -0,0 +1,47 @@
+@import "../utils/variables";
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, 
+  minmax($block-min-width, $block-max-width));
+
+  @media (max-width: 899px) {
+    grid-template-columns: repeat(2, 
+    minmax($block-min-width, $block-max-width));
```

так переносить нельзя
```suggestion
    grid-template-columns: repeat(2, minmax($block-minwidth, $block-max-width));
```